src/reducers/shell.js

```diff
@@ -84,6 +91,7 @@ export const actions = {
   addPlayerClass: (className: string) => ({type: types.ADD_PLAYER_CLASS, className}),
   removePlayerClass: (className: string) => ({type: types.REMOVE_PLAYER_CLASS, className}),
   updateIsMobile: (isMobile: boolean) => ({type: types.UPDATE_IS_MOBILE, isMobile}),
+  updatePlayerSize: (playerSize: string) => ({type: types.UPDATE_PLAYER_SIZE, playerSize}),
```

Comment:
```suggestion
  updatePlayerSize: (playerSize: number) => ({type: types.UPDATE_PLAYER_SIZE, playerSize}),
```

Reply:
leaving as is, as the playerSize will be string